engines/cryo/cryo.h

```diff
@@ -82,6 +83,10 @@ class CryoEngine : public Engine {
 	void getMousePosition(int16 *x, int16 *y);
 	void setMousePosition(int16 x, int16 y);
 	bool isMouseButtonDown();
+
+private:
+	int _mouseButton;
+	// byte _keyState[256];
```

Comment:
Please remove

Reply:
Should I also remove the reference to `_keyState` in `CryoEngine::pollEvents()` as well?

Reply:
Yes please. The keyup events should be handled by the event manager... it seems that the _keyState array is a leftover

Reply:
OK, done.